templates/bundler_audit.rake

```diff
@@ -4,9 +4,25 @@ if Rails.env.development? || Rails.env.test?
   namespace :bundler do
     desc "Updates the ruby-advisory-db and runs audit"
     task :audit do
+      prepare_bundler_audit_to_exit_with_error_if_update_fails
       %w(update check).each do |command|
         Bundler::Audit::CLI.start [command]
       end
```

Comment:
> It could be: `Bundler::Audit::CLI.start ["check"]`. Where's a good spot to put the call to `update`?

Don't we already do it in `if Bundler::Audit::Database.update!`?